content/en/examples/policy/priority-class-resourcequota.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: pods-cluster-services
+spec:
+  scopeSelector:
+    matchExpressions:
+      - operator : In
+        scopeName: PriorityClass
+        values: ["cluster-services"]
```

Comment:
I would add a newline at the end of this manifest so that it looks OK no matter what OS you download it on.